repo/packages/A/arangodb/0/config.json

```diff
@@ -0,0 +1,86 @@
+{
+  "type": "object",
+  "properties": {
+
+    "mesos/master": {
+      "description": "The URL of the Mesos master. The format is a comma-delimited list of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard URLs like http://localhost are also acceptable.",
+      "type": "string",
+      "default": "zk://master.mesos:2181/mesos"
+    },
+
+    "arangodb/id": {
```

Comment:
we would like these defined in hierarchies and have a preference for dot notation.  Please look at: https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/M/marathon/1/config.json
as an example.   defining `arangodb/` for each property is not the preference here.